save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/ChildrenBuilderUtils.kt

```diff
@@ -126,13 +126,16 @@ private fun ChildrenBuilder.buttonBuilder(
     button {
         type = ButtonType.button
         val builtClasses = buildString {
-            append("btn btn-")
-            if (isOutline) {
-                append("outline-")
-            }
-            append(style)
-            if (isActive) {
-                append(" active")
+            append("btn")
+            style?.let { style ->
```

Comment:
maybe use `it` or `style` directly without shadowing

Reply:
Yeah, sure

Reply:
Done